lib/matplotlib/tests/test_colors.py

```diff
@@ -715,3 +716,34 @@ def __add__(self, other):
         else:
             assert len(recwarn) == 0
         recwarn.clear()
+
+
+def _brighten_test_helper(color, shade, expected):
+    sc = mcolors.brighten_color(color, shade)
+    assert_equal(sc, expected)
+
+
+def test_color_brightening():
+    test_colors = (
+        'red',
+        'red',
+        'red',
+        'red',
+        'red',
+        [0, .5, .9],
+        'slategrey',
+    )
+    test_brighten = (0, 0.50, 1.00, -0.50, -1.00, 0.20, -0.20)
+    known_brighten_result = (
+        (1.0, 0.0, 0.0),
+        (1.0, 0.5, 0.5),
+        (1.0, 1.0, 1.0),
+        (0.5, 0.0, 0.0),
+        (0.0, 0.0, 0.0),
+        (0.080000000000000071, 0.59111111111111092, 1.0),
+        (0.35097730430754981, 0.40156862745098038, 0.45215995059441105)
+    )
+    for color, brighten, expected in zip(test_colors,
+                                      test_brighten,
+                                      known_brighten_result):
+        _brighten_test_helper(color, brighten, expected)
```

Comment:
I think we're missing the obvious case that this could just be written:

```
_brighten_test_helper('red', 0, (1.0, 0.0, 0.0))
_brighten_test_helper('red', 0.5, (1.0, 0.5, 0.5))
...
```

These aren't test parameters - they're the tests themselves, and the loops just obscure the pairing between the results and the expected values